lab-doug/test/showContents-test.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var expect = require('chai').expect;
+var showContents = require(__dirname + '/../lib/showContents');
+
+describe('Testing showContents module', function(){
+  describe('Testing showContents()', function(){
+    it('should return "hello bob"', function(){
```

Comment:
That's not actually what you expect it to do--be careful to name your tests consistently with what you expect them to do.